arch/risc-v/src/mpfs/mpfs_fpga_canfd.c

```diff
@@ -735,19 +784,16 @@ static void mpfs_receive_work(void *arg)
         MPFS_CANFD_RX_STATUS_RXFRC) >> MPFS_CANFD_RX_STATUS_RXFRC_SHIFT;
     }
 
-  /* Check for RX FIFO Overflow */
```

Comment:
The original way of checking the DOR should work just fine here. There is no need to add the isr_status to priv->can in that case. I will still try that out